### ISSUE TITLE
Graphic for device agent docs

### ISSUE THREAD

**Issue by Yndira-E** [FlowFuse/website#2670]
### Description

**Considerations:**
- Where will the image be displayed/shared, product? blog post? social media? presentation? website?

**Inspirations:**
- <link here>
- <link here>

**Comment by Yndira-E** [comment]
Hi @joepavitt wdyt?

![Image](https://github.com/user-attachments/assets/d28d396e-f06a-4b47-8ec3-032c5f8db589)

**Comment by joepavitt** [comment]
Feel like we need clearer labelling on the high level steps, e.g. "Installation" & "Deploy Flows". Maybe the bottom row could be a right-side instead? Left-side is "Install", right-side becomes "Deploy"?

**Comment by Yndira-E** [comment]
Yeah, that's how you lay it out initially. However, I was trying to avoid making it too long horizontally, as it could become unreadable on smaller screens.

I'll see what I can do.

**Comment by joepavitt** [comment]
Could we split the bottom row into two blue sections? Both labelled Deployment, but with Option 1/2 in them? Think that's the bit that's reads as confusing at the moment.

**Comment by Yndira-E** [comment]
> Could we split the bottom row into two blue sections?

Done!

![Image](https://github.com/user-attachments/assets/03b19d6e-2388-4d0c-8b72-9adbdf9688c5)

**Comment by joepavitt** [comment]
Thanks @Yndira-E - I do still thinking the bottom should go to the right, as the general flow reads (to start with) left > right, then down, left left > right again.

I checked how it would render in the docs themselves, and this was the outcome:

<img width="1718" alt="Screenshot 2024-10-22 at 13 11 11" src="https://github.com/user-attachments/assets/dc0117f7-a2cd-480f-96e1-f7caf5575776">

**Comment by Yndira-E** [comment]
I thought a `max-width` would be set and `data-zoomable` added. On mobile it should be fine taking up  the full width.

Here's the horizontal version in case you'd prefer to use this one for desktop instead.
![Image](https://github.com/user-attachments/assets/b8a1c30b-f6ca-40ef-b741-9b8c3886fcf2)

**Comment by Yndira-E** [comment]
Does the iteration on this section involve more than the graphic? I’ve put the device agent post on hold until we’ve made improvements.

**Comment by joepavitt** [comment]
> Does the iteration on this section involve more than the graphic? I’ve put the device agent post on hold until we’ve made improvements.

It'll be an upgrade of section alongside the graphic I think, not 100%

**Comment by Yndira-E** [comment]
> But why make the image intentionally smaller and have load of empty white-space on the right?

> Here's the horizontal version in case you'd prefer to use this one for desktop instead.
> ![Image](https://github.com/user-attachments/assets/668ad447-c211-4fec-b278-c53ee9de4354)

I trust your judgment—pick the one you think looks best for each screen size. Just let me know when this section of the docs is updated so we can review the user flow from the `/device-agent` page and, finally, promote it on socials 😉
